src/lwc/alertBanner/alertBanner.js

```diff
@@ -0,0 +1,78 @@
+import { LightningElement, api } from 'lwc';
+import { isEmpty } from 'c/utilCommon';
+
+const defaultClass = 'slds-notify slds-notify_alert slds-theme_alert-texture slds-theme_';
```

Comment:
Could we update this to uppercase `DEFAULT_CLASS` or something along those lines?